tools/hclust2/hclust2.xml

```diff
@@ -0,0 +1,562 @@
+    <description>plots heatmaps</description>
+    <macros>
+        <token name="@VERSION@">0.99</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@VERSION@">hclust2</requirement>
+    </requirements>
+    <command detect_errors="exit_code">
+    <![CDATA[
+    hclust2.py
+	    -i '$i'
+	    #if $o == "png"
+			-o 'output.png'
+		#else if $o == "pdf"
+			-o 'output.pdf'
+		#else if $o == "svg"
+			-o 'output.svg'
+		#end if
+	    --legend_file 'legend.png'
+		#if $inp_data_mat_param.out_table
+	        --out_table '$out_table'
+	    #end if
+		--fname_row '$inp_data_mat_param.fname_row'
+	    --sname_row '$inp_data_mat_param.sname_row'
+	    --fperc '$inp_data_mat_param.fperc'
+	    --sperc '$inp_data_mat_param.sperc'
+	    #if  str($inp_data_mat_param.metadata_rows) != ''
+	        --metadata_rows '$inp_data_mat_param.metadata_rows'
+	    #end if
+	    #if str($inp_data_mat_param.skip_rows) != ''
+	        --skip_rows '$inp_data_mat_param.skip_rows'
+	    #end if
+	    #if str($inp_data_mat_param.ftop) != ''
+	        --ftop '$inp_data_mat_param.ftop'
+	    #end if   	
+	    #if str($inp_data_mat_param.stop) != ''
+	        --stop '$inp_data_mat_param.stop'
+	    #end if
+		#if str($inp_data_mat_param.def_na) != ''
+	        --def_na '$inp_data_mat_param.def_na'
+	    #end if    
+		--f_dist_f '$dist_param.f_dist_f'
+	    --s_dist_f '$dist_param.s_dist_f'
+		#if str($dist_param.load_dist_f_select.load_dist_f_selector) == "yes"
+	        --load_dist_matrix_f '$dist_param.load_dist_f_select.load_dist_matrix_f'
+	    #end if
+	    #if str($dist_param.load_dist_s_select.load_dist_s_selector) == "yes"
+	        --load_dist_matrix_s '$dist_param.load_dist_s_select.load_dist_matrix_s'
+	    #end if
+	    #if str($dist_param.load_pickled_dist_f_select.load_pickled_dist_f_selector) == "yes"
+	        --load_pickled_dist_matrix_f '$dist_param.load_pickled_dist_f_select.load_pickled_dist_matrix_f'
+	    #end if
+	    #if str($dist_param.load_pickled_dist_s_select.load_pickled_dist_s_selector) == "yes"
+	        --load_pickled_dist_matrix_s '$dist_param.load_pickled_dist_s_select.load_pickled_dist_matrix_s'
+	    #end if
+		#if $dist_param.save_pickled_dist_matrix_f
+	        --save_pickled_dist_matrix_f '$save_pickled_dist_matrix_f'
+	    #end if
+	    #if $dist_param.save_pickled_dist_matrix_s
+	        --save_pickled_dist_matrix_s '$save_pickled_dist_matrix_s'
+	    #end if
+	    $clust_param.no_fclustering
+		$clust_param.no_sclustering        			
+	    --flinkage '$clust_param.flinkage'
+	    --slinkage '$clust_param.slinkage'
+	    #if str($heatmap_opt.title) != ''
+	        --title '$heatmap_opt.title'
+	    #end if
+	    --dpi '$heatmap_opt.dpi'
+	    $heatmap_opt.log_scale
+	    $heatmap_opt.sqrt_scale
+	    $heatmap_opt.no_flabels 
+		$heatmap_opt.no_slabels 
+	    #if str($heatmap_opt.minv) != ''
+	        --minv '$heatmap_opt.minv'
+	    #end if
+	    #if str($heatmap_opt.maxv) != ''
```

Comment:
`#if str($heatmap_opt.maxv) != ''` should be the same as ` #if str($heatmap_opt.maxv)`